server/svix-server/src/error.rs

```diff
@@ -16,35 +16,59 @@ use serde_json::json;
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// The error type returned from the Svix API
-#[derive(Debug, Clone)]
-pub enum Error {
-    /// A generic error
-    Generic(String),
-    /// Database error
-    Database(String),
-    /// Queue error
-    Queue(String),
-    /// Database error
-    Validation(String),
-    /// Any kind of HttpError
-    Http(HttpError),
+#[derive(Debug)]
```

Comment:
Not a big concern, but how does the debug print look like in the terminal? It might make more sense to implement `std::fmt::Debug` instead of deriving it?

Should there be alternative formats depending on log format?

Again, not super important, at least for the first draft.